bin/i18n/resources/pegasus/emails/sync_in.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/sync_in_base'
+require_relative '../../../utils/pegasus_markdown'
+require_relative '../emails'
+
+module I18n
+  module Resources
+    module Pegasus
+      module Emails
+        class SyncIn < I18n::Utils::SyncInBase
+          LOCALIZABLE_FILE_SUBPATHS = %w[
+            hoc_signup_2023_receipt.md
+          ].freeze
+
+          def process
+            LOCALIZABLE_FILE_SUBPATHS.each do |file_name|
+              i18n_source_file_path = File.join(I18N_SOURCE_DIR_PATH, file_name)
+              I18nScriptUtils.copy_file(File.join(ORIGIN_DIR_PATH, file_name), i18n_source_file_path)
+              I18n::Utils::PegasusMarkdown.sanitize_file_header(i18n_source_file_path)
```

Comment:
`I18n::Utils::PegasusMarkdown.sanitize_file_header` removes every header attr except `title`.
I believe we would also want to localize email `subject`. https://github.com/code-dot-org/code-dot-org/blob/47a5b0be4eaf78f6b9c250a1b2c80e91aad1d74e/pegasus/emails/hoc_signup_2023_receipt.md?plain=1#L4

Reply:
Please create a new util for this based on `I18n::Utils::PegasusMarkdown`:

1. Change `module PegasusMarkdown` to `class PegasusMarkdown` in `bin/i18n/utils/pegasus_markdown.rb`
2. Create new util `bin/i18n/utils/pegasus_email.rb`:
```ruby
require_relative 'pegasus_markdown'

module I18n
  module Utils
    class PegasusEmail < I18n::Utils::PegasusMarkdown
      def self.sanitize_header(header)
        header.slice('subject')
      end
    end
  end
end
```

```suggestion
              I18n::Utils::PegasusEmail.sanitize_file_header(i18n_source_file_path)
```

Reply:
Will do! Thank you for the detailed feedback!